builds/jenkins-delete-resources/Jenkinsfile

```diff
@@ -0,0 +1,395 @@
+import groovy.json.JsonSlurper
+import groovy.transform.Field
+import groovy.json.JsonOutput
+
+@Field def inputParams
+@Field def config
+@Field def awsAccounts
+@Field def finalAccounts = []
+@Field def credentialId
+@Field def isPrimary = false
+
+
+// To be replaced as @Field def repo_credential_id = "value" for repo_credential_id, repo_base and repo_core
+@Field def repo_credential_id 
+@Field def aws_credential_id 
+@Field def region 
+@Field def instance_prefix 
+@Field def repo_base 
+@Field def repo_core
+@Field def scm_type 
+
+@Field def configModule
+@Field def configLoader
+
+
+node {
+ echo "Starting the Job with Params ${params}"
+ inputParams = params.input
+ // Load the config
+ def jazzBuildModuleURL = getBuildModuleUrl()
+ loadBuildModules(jazzBuildModuleURL)
+ def output = JsonOutput.toJson(configLoader)
+ echo "${output}"
```

Comment:
Remove unwanted consoles here and other places of this PR

Reply:
Removed.